src/ui/components/pages/swap/form.tsx

```diff
@@ -410,59 +409,58 @@ export function SwapForm({
     }
   }, [swapVendorTouched, profitVendor]);
 
+  const swappableAssetEntries = usePopupSelector(
```

Comment:
Maybe rename it to `swappableAssetIdsByVendor`?
The current name is a bit confusing.

Reply:
I would also say that redux state and storage key should be renamed as well